README.md

```diff
@@ -1 +1,13 @@
-# esb-connector-cerner
\ No newline at end of file
+# Cerner ESB Connector
+
+The Cerner connector allows you to access the cerner FHIR APIs of cerner Millennium EHR platform through  WSO2 EI. Cerner connector currently supports the DSTU 2 Final (1.0.2) version of the FHIR standard.
+
+### Build
+
+mvn clean install
+
+### How You Can Contribute
+
+You can create a third party connector and publish in WSO2 Store.
+
+https://docs.wso2.com/display/ESBCONNECTORS/Creating+a+Third+Party+Connector+and+Publishing+in+WSO2+Store
```

Comment:
Please use correct link as this is broken. 
Check https://docs.wso2.com/display/ESBCONNECTORS/Creating+and+Publishing+a+Third+Party+Connector